### DISCUSSION TITLE
In a custom theme, how do I change the line number color?

### DISCUSSION THREAD

**Discussion by talwat** [sharkdp/bat#2481] [Q&A]
Pretty simple, I want this because it's very hard to read currently. After a lot of fussing with the color picker, it gave me `#3f3f3f` which isn't defined anywhere in the theme file.

Theme: [pastebin](https://pastebin.com/yZRBXscF)

<img width="434" alt="Screen Shot 2023-02-22 at 10 35 36" src="https://user-images.githubusercontent.com/83217276/220580561-0ced0c4b-3158-4b5c-8e95-64713334d350.png">

**Comment by keith-hall** [comment]
did you try setting `gutterForeground` in the theme settings? https://github.com/trishume/syntect/blob/master/src/highlighting/theme.rs#L83
otherwise `bat` defaults to a fixed color:
https://github.com/sharkdp/bat/blob/master/src/printer.rs#L706-L713
https://github.com/sharkdp/bat/blob/master/src/printer.rs#L686

**Reply by talwat** [reply]
Yep! That did the trick!

Just one more question, what does `gutter` do? I tried setting it to a very bright value but it didn't appear.

**Reply by keith-hall** [reply]
`bat` doesn't use the `gutter` color for anything. In Sublime Text and other editors it would affect the background color of the area on the left where the line numbers appear. Which kinda makes sense, as gutter foreground controls the line numbers :)
